qa/tasks/cbt_performance.py

```diff
@@ -50,12 +50,18 @@ def collect(self, ctx, config):
                     "benchmark" : benchmark["benchmarks"],
                     "results" : cbt_results.get("results", None),
                 }
-                response = requests.post(self.endpoint_url, json=data, headers=self.headers, auth=self.auth)
-                if response.status_code == 201:
-                    self.log.info("Data inserted successfully.")
-                    ctx.summary['cbt_perf_url'] = self.create_cbt_perf_url(ctx, config)
-                else:
-                    self.log.info(f"Error inserting data: {response}")
+
+                try:
+                    response = requests.post(self.endpoint_url, json=data, headers=self.headers, auth=self.auth, timeout=10)
+                    if response.status_code == 201:
+                        self.log.info("Data inserted successfully.")
+                        ctx.summary['cbt_perf_url'] = self.create_cbt_perf_url(ctx, config)
+                    else:
+                        self.log.info(f"Error inserting data: {response}")
+                except requests.exceptions.RequestException as e:
+                    self.log.warning(f"Could not send performance data to {self.endpoint_url}: {e}")
+                    self.log.warning("Continuing without performance data collection")
```

Comment:
Thanks for catching this! Any idea why we are unable to send data consistently?
See: https://pulpito.ceph.com/yuriw-2025-10-21_18:47:17-crimson-rados-tentacle-release-distro-crimson-debug-smithi/

Can this be backported to T?

Reply:
mira118 was shutdown, we can't use it anymore, and yes, i'll backport it to T as well